src/dev/flang/util/SourceFile.java

```diff
@@ -807,10 +807,13 @@ public int lineEndPos(int line)
    */
   public int codePointInLine(int pos, int line)
   {
+    if(PRECONDITIONS)
```

Comment:
please add a space between `if` and `(`. 

Also, the form I use for preconditions is
```
if (PRECONDITIONS) require
  (line >= 0);
```
this is to make this look more similar to Eiffel's `requires` or Fuzion's `pre` clause:
```
  pre
     line >= 0
```
please fix in separate PR.